cfgov/jinja2/v1/events/_macros.html

```diff
@@ -291,3 +315,79 @@ <h2 class="event-venue_heading">{{ city }}</h2>
    {%- endfor -%}
 {%- endfor -%}
 {% endmacro %}
+
+{# ==========================================================================
+
+   event_media()
+
+   ==========================================================================
+
+   Description:
+
+   Create event archive media markup when given:
+
+   event: An event from a query result.
+
+   ========================================================================== #}
+
+{% macro event_media( event ) %}
+{% set col_classes = 'content-l_col
+                      content-l_col-1-2
+                      block
```

Comment:
Avoid mixing `content-l` and `block` classes. I've learned the hard way this can end up w/ unexpected bugs/layout changes later on when CF is updated. I'd place the `block` inside the columns